### ISSUE TITLE
URIs in example 1 (part 2 CRS)

### ISSUE THREAD

**Issue by heidivanparys** [opengeospatial/ogcapi-features#365]
None of the following URIs in example 1 seem to return a valid CRS definition, an exception report is returned instead. Although it is "only" an example, it would still be good to have an example that is as correct is possible.

I guess that in URIs 2 to 4, the only change that should be done is changing OGC to EPSG. I don't know what was meant with the 1st or the 5th URI.

```
          "http://www.opengis.net/def/crs/OGC/1.3/CRS41001",
          "http://www.opengis.net/def/crs/OGC/0/2246",
          "http://www.opengis.net/def/crs/OGC/0/3130",
          "http://www.opengis.net/def/crs/OGC/0/3634",
          "http://www.opengis.net/def/crs/OGC/0/6702",
```

**Comment by nmtoken** [comment]
I guess for 5 the following is meant:

    http://www.opengis.net/def/datum/EPSG/0/6702

and for 1 it's possibly

    http://www.opengis.net/def/crs/OGC/1.3/AUTO42001

**Comment by cportele** [comment]
Meeting 2020-04-27: The comment is correct, the URIs need to be checked and updated. All URIs will be validated before the approval vote.

**Comment by cportele** [comment]
Addressed by #395 

Meeting 2020-05-25: Merge the change requests to create a consolidated readable draft document with all edits and distribute for review in the SWG. We can reopen issues, where needed.
